homework-submission/routing/src/components/DynamicToDoList/DynamicToDoList.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import './index.css';
+import TodoList from './TodoList';
+
+class DynamicToDoList extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
No need for the extra div element, this could be written as `return <TodoList />;`